THIRD-PARTY-Missed-Licenses.txt

```diff
@@ -0,0 +1,73 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+#-------------------------------------------------------------------------------
+# Already used licenses in project :
+# - ASL, version 2
+# - Apache 2
+# - Apache 2.0
+# - Apache 2.0 License
+# - Apache License 2.0
+# - Apache License V2.0
+# - Apache License Version 2.0
+# - Apache License, 2.0
+# - Apache License, Version 2.0
+# - Apache License, version 2.0
+# - Apache Software License, version 1.1
+# - Apache Software License, version 2.0
+# - Apache-2.0
+# - BSD
+# - BSD 2-Clause
+# - BSD 2-Clause License
+# - BSD License 3
+# - BSD-2-Clause
+# - CDDL
+# - CDDL + GPLv2 with classpath exception
+# - CDDL 1.1
+# - CDDL License
+# - CDDL v1.1 / GPL v2 dual license
+# - COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0
+# - Common Development and Distribution License (CDDL) v1.0
+# - Common Public License Version 1.0
+# - Dual license consisting of the CDDL v1.1 and GPL v2
+# - EDL 1.0
+# - EPL 2.0
+# - Eclipse Public License - v 1.0
+# - Eclipse Public License - v 2.0
+# - Eclipse Public License 1.0
+# - GNU General Public Library
+# - GNU Lesser General Public License
+# - GNU Lesser General Public License, Version 2.1
+# - GNU Lesser Public License
+# - GPL2 w/ CPE
+# - GPLv2+CE
+# - Indiana University Extreme! Lab Software License, vesion 1.1.1
+# - LGPL
+# - LGPL 2.1
+# - LGPL, version 2.1
+# - Lesser General Public License, version 3 or greater
+# - MIT License
+# - MIT license
+# - MPL 1.1
+# - Modified BSD
+# - Mozilla Public License 1.1 (MPL 1.1)
+# - Mozilla Public License, Version 2.0
+# - New BSD License
+# - Public Domain
+# - Public Domain, per Creative Commons CC0
+# - The 2-Clause BSD License
+# - The Apache License, Version 2.0
+# - The Apache Software License, Version 2.0
+# - The GNU General Public License (GPL), Version 2, With Classpath Exception
+# - The MIT License
+# - The PostgreSQL License
+# - W3C license
+# - jQuery license
+#-------------------------------------------------------------------------------
+# Please fill the missing licenses for dependencies :
+#
+#
+#Thu Feb 25 15:59:34 EET 2021
+javax.xml.bind--jaxb-api--2.1=
+com.sun.xml.bind--jaxb-xjc--2.1.6=
+com.sun.xml.bind--jaxb-impl--2.1.6=
+xerces--xmlParserAPIs--2.6.2=
+com.google.code.javaparser--javaparser--1.0.10=
```

Comment:
This file and ` src/license/THIRD-PARTY.properties` are the same. Please remove the duplicate this is not used.